spec/lib/faraday/zipkin-tracer_spec.rb

```diff
@@ -17,29 +17,29 @@ def on_complete
   end
 
   # Ruby 1.8 didn't support \h
-  HEX_REGEX = RUBY_VERSION >= "1.9.2" ? '\h' : '[0-9a-fA-F]'
+  HEX_REGEX = RUBY_VERSION >= '1.9.2' ? '\h' : '[0-9a-fA-F]'
```

Comment:
since the syntax uses 1.9 hash syntax this can be removed altogether in favor of say
```ruby
HEX_REGEX = /\A\h{16}\z/
```
(always found it odd to have the var called *_regex but it being actually a string)